lte/gateway/python/magma/pipelined/tests/test_inout_non_nat.py

```diff
@@ -525,25 +557,33 @@ def setUpClass(cls):
         launch the ryu apps for testing pipelined. Gets the references
         to apps launched by using futures.
         """
-        inout.getmacbyip6 = mocked_getmacbyip6
-        inout.get_mobilityd_gw_info = mocked_get_mobilityd_gw_info
-        inout.set_mobilityd_gw_info = mocked_set_mobilityd_gw_info
+        gw_mac_address.getmacbyip6 = mocked_getmacbyip6
```

Comment:
It's not a good idea to overwrite the imports of modules manually, this should probably be done via an `@patch` for the test.